extra/release_tool.py

```diff
@@ -153,6 +153,7 @@ def get_components_of_type(
         only_non_release=False,
         only_independent_component=False,
         only_non_independent_component=False,
+        only_core_components=True,
```

Comment:
By looking at all places we had to set `only_core_components=False` to call this, should we reconsider the default value?

Reply:
We could, I do like the explicitness though

Reply:
okey, no strong feelings here.